task/06-conditions-n-loops-tasks.js

```diff
@@ -373,7 +461,26 @@ function toNaryString(num, n) {
  *   ['/web/favicon.ico', '/web-scripts/dump', '/webalizer/logs'] => '/'
  */
 function getCommonDirectoryPath(pathes) {
-    throw new Error('Not implemented');
+    let str = '';
+    for (let i = 0; i < pathes[0].length; ++i) {
+        str += pathes[0][i];
+        let c = 0;
```

Comment:
if c has only values of 1 and 0 use boolean type. Also use better name for the variable